disnake/ext/commands/base_core.py

```diff
@@ -571,50 +569,31 @@ async def can_run(self, inter: ApplicationCommandInteraction) -> bool:
             inter.application_command = original
 
 
-# kwargs are annotated as None to ensure the user gets a linter error when using them
-def guild_permissions(
-    guild_id: int,
-    *,
-    roles: Optional[Mapping[int, bool]] = None,
-    users: Optional[Mapping[int, bool]] = None,
-    owner: Optional[bool] = None,
-    **kwargs: None,
-) -> Callable[[T], T]:
+def default_member_permissions(**permissions: Literal[True]) -> Callable[[T], T]:
```

Comment:
we should allow this to take a value, just like `disnake.Permissions`

Reply:
531d34a787ce9dc859c1858656bacedda4a1cae9